src/test/java/com/shinesolutions/aemorchestrator/service/AemInstanceHelperServiceTest.java

```diff
@@ -552,14 +546,14 @@ public void testGetDispatcherIdForPairedPublishWithFoundPair() throws Exception
     }
     
     @Test
-    public void testGetDispatcherIdForPairedPublishWithNoPair() throws Exception {
+    public void testGetDispatcherIdForPairedPublishWithNoPair() {
         String instance1 = "1st-876543";
-        Map<String, String> tagsMissingPair = new HashMap<String, String>();
+        Map<String, String> tagsMissingPair = new HashMap<>();
         
-        List<String> instanceIds = new ArrayList<String>();
+        List<String> instanceIds = new ArrayList<>();
         instanceIds.add("1st-876543");
 
-        when(awsHelperService.getInstanceIdsForAutoScalingGroup(envValues.getAutoScaleGroupNameForPublish())).thenReturn(instanceIds);
```

Comment:
`envValues.getAutoScaleGroupNameForPublish()` here did not match the `envValues.getAutoScaleGroupNameForPublish()` in the `getInstanceIdsForAutoScalingGroup` method. Hence I replaced it with `anyString()`
@cliffano